packages/expo-image/src/Image.types.ts

```diff
@@ -242,13 +242,25 @@ export interface ImageProps extends ViewProps {
   accessible?: boolean;
 
   /**
-   * The text that's read by the screen reader when the user interacts with the image.
+   * The text that's read by the screen reader when the user interacts with the image. Sets the the `alt` tag on web which is used for web crawlers and link traversal.
    * @default undefined
    * @platform android
    * @platform ios
+   * @platform web
    */
   accessibilityLabel?: string;
 
+  /**
+   * The text that's read by the screen reader when the user interacts with the image. Sets the the `alt` tag on web which is used for web crawlers and link traversal.
```

Comment:
```suggestion
   * The text that's read by the screen reader when the user interacts with the image. Sets the the `alt` tag on web which is used for web crawlers and link traversal. Is an alias for `accessibilityLabel`
```

Think this might make it clearer that there's no difference between the two. WDYT @EvanBacon?